packages/blade/src/components/Checkbox/CheckboxLabelText.tsx

```diff
@@ -0,0 +1,16 @@
+import BaseText from '../Typography/BaseText';
+
+const CheckboxLabelText = ({ children }: { children: React.ReactNode }): React.ReactElement => {
+  return (
+    <BaseText
```

Comment:
can't we directly use this in checkbox as label instead of creating a new LabelText component?

Reply:
Yeah we can. Separated it for readability reasons only. Since now if we get the <Text /> I'll inline it.